CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+##Supported Release 1.4.1
```

Comment:
this is done through out the changelog, should i fix it all or follow what is there?